x-pack/plugins/security_solution/public/detections/components/rules/step_about_rule/data.tsx

```diff
@@ -22,32 +22,38 @@ const StyledEuiHealth = styled(EuiHealth)`
   line-height: inherit;
 `;
 
+export const RISK_COLOR_LOW = euiLightVars.euiColorVis0;
+export const RISK_COLOR_MEDIUM = euiLightVars.euiColorVis5;
+export const RISK_COLOR_HIGH = euiLightVars.euiColorVis7;
+export const RISK_COLOR_CRITICAL = euiLightVars.euiColorVis9;
+
+export const RISK_SCORE_LOW = 21;
```

Comment:
Can you add a comment just detailing why the values are these. We may want to reference this for the docs team as well

Reply:
> Can you add a comment just detailing why the values are these. We may want to reference this for the docs team as well

😂 I didn't define or change these values, so I can't speak to, or document why for example `low` is `21`.

This PR merely takes the previous definition of these values:

```ts
export const defaultRiskScoreBySeverity: Record<Severity, number> = {
  low: 21,
  medium: 47,
  high: 73,
  critical: 99,
};
```

and exposes them as constants for use in other files.